Krypt/Source/KVConnectSMIME.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  KVConnect.swift
+//  CryptoSwift
+//
+//  Created by Miso Lubarda on 29.04.19.
+//
+
+import Foundation
+
+public struct KVConnectSMIME {
```

Comment:
```suggestion
public struct KVConnect {
```
I think `SMIME` is implementation detail, should be a blackbox for the consumer. Wyt?

Reply:
I think we should do it the other way arround: this is generic SMIME to MIME decryption and verification. It has nothing to do with KVConnect. We should rename everything to SMIMEDecryptor or something similar. What do you say?